### ISSUE TITLE
bandstats band numbers don't correlate to vasp bands number

### ISSUE THREAD

**Issue by badw** [SMTG-UCL/sumo#36]
The band indices don't correspond to the band indices outputted by VASP. I appreciate this is a python thing starting from `0` etc. but it would be nice if the `sumo-bandstats` screen output corresponded to the VASP output.

### From `sumo-bandstats`:
```
Direct band gap: 0.900 eV
k-point: [0.00, 0.00, 0.50]
k-point indexes: 190, 191, 359
Band indexes: 55, 56
```

### From EIGENVAL:
```
.
.
.

56        4.887403   1.000000
57        6.206457   0.000000
.
.
.



```

**Comment by ajjackson** [comment]
You found our hidden scheme to educate people in the superiority of zero-indexing :stuck_out_tongue_closed_eyes: 

This is probably going to confuse some people, we should either change it or add a note to inform people it is zero-indexed.

This would be a good job for a novice contributor. Some changes need to be made to the `_log_band_gap_information` function in _sumo/cli/bandstats.py_. Specifically, the index values need to be changed before they are converted to strings for logging.

While at it, they could also change "indexes" to "indices"!

**Comment by utf** [comment]
Fixed in  928aabecf3cd776b9f30372817c8afc95926db12.
- band index = band index+1
- indexes -> indices
